raiden/tests/scenarios/bf5_join_and_leave.yaml

```diff
@@ -0,0 +1,165 @@
+      - parallel:
+          name: "Assert after channel openings"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - parallel:
+          name: "Deposit in the other direction"
+          tasks:
+            - deposit: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, expected_http_status: 200}
+      - parallel:
+          name: "Assert after deposits"
+          tasks:
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Node2 joins the network"
+          tasks:
+            # It should not matter that `initial_channel_target` is 3 since there are only two nodes in the network
+            - join_network: {from: 2, funds: 1_000_000_000_000_000_000, initial_channel_target: 3, joinable_funds_target: 0.4, expected_http_status: 204}
+      - parallel:
+          name: "Assert after node2 joined"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            # Deposits from node2 should be funds / 5
+            - assert: {from: 2, to: 0, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "Node3 joins the network"
+          tasks:
+            - join_network: {from: 3, funds: 1_000_000_000_000_000_000, initial_channel_target: 3, joinable_funds_target: 0.4, expected_http_status: 204}
+      - parallel:
+          name: "Assert after node3 joined"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 0, to: 3, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000, balance: 1_000_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 3, total_deposit: 0, balance: 0, state: "opened"}
+            # Deposits from node2 should be funds / 5
+            - assert: {from: 2, to: 0, total_deposit: 200_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
```

Comment:
Is there any way to do these calculations inside the yaml file? Use a variable for the initial deposit and here have it divided by 5?